src/predict/jobs/forecast/models.py

```diff
@@ -29,7 +27,7 @@ def fetch_bigquery_data():
     @staticmethod
     def fetch_health_tips():
         "fetch health tips from the api"
-        response = requests.get(Events.events_tips_url, headers={"authorization": configuration.AIRQO_API_AUTH_TOKEN})
+        response = requests.get(Events.events_tips_url, params={"TOKEN": configuration.AIRQO_API_AUTH_TOKEN})
```

Comment:
Thanks @Mnoble-19 
I think the query parameter should be lower case, that is: `......params = {"token": configuration.AIRQO_API_AUTH_TOKEN}`

Reply:
@Mnoble-19 
Additionally, please follow up with the update of the ENVs for both `production` and `staging`. @123MwanjeMike

Reply:
> @Mnoble-19 Additionally, please follow up with the update of the ENVs for both `production` and `staging`. @123MwanjeMike

Yes this fine. I have access to secret manager and have done so accordingly

Reply:
> Thanks @Mnoble-19 I think the query parameter should be lower case, that is: `......params = {"token": configuration.AIRQO_API_AUTH_TOKEN}`

Done